package.json

```diff
@@ -12,6 +12,36 @@
     "TailWind CSS"
   ],
   "main": "tailwind.config.js",
+  "types": "tailwind.config.d.ts",
```

Comment:
Add typescript types.

Reply:
They are just a re-export of tailwindcss's types but it makes my package manager and editor happy.